superset/assets/src/SqlLab/components/SqlEditor.jsx

```diff
@@ -314,7 +313,7 @@ class SqlEditor extends React.PureComponent {
         onDragStart={this.onResizeStart}
         onDragEnd={this.onResizeEnd}
       >
-        <div ref={this.northPaneRef}>
+        <div ref={this.northPaneRef} className="NorthPane">
```

Comment:
i know we haven't been overly consistent with css classes, but i most commonly see `camelCase` or `kebab-case` in Superset, so let's use one of those? (instead of adding another style `PascalCase`)

Reply:
We use `PascalCase` sometimes when we want to identify a React component directly (1:1 mapping). Otherwise generally `kebab-case`

Reply:
Thanks for catching this. I've corrected this to `kebab-case`, and will keep an eye out for this in the future.